corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java

```diff
@@ -41,8 +41,9 @@
         "[--diskPath=<pathToTempDirForLargeTables>] "+
         "[--numItems=<numItems>] "+
         "[--batchSize=<itemsPerTransaction>] "+
-        "[--itemSize=<sizeOfEachRecordValue>] "+
-        "[--tlsEnabled=<tls_enabled>]\n"
+        "[--itemSize=<sizeOfEachRecordValue>] "
```

Comment:
Is it correct to think of renaming this file? as we are differentiating browser from editor, so its more like CorfuStoreBrowserEditorMain?